packages/components/Callout/macos/RCTCalloutManager.h

```diff
@@ -0,0 +1,4 @@
+#import <React/RCTTouchHandler.h>
```

Comment:
Why are all of these in the header file? Can we just move them out to the .mm where they're needed? Also no interface needs to be defined here?

Reply:
There is no .mm file, there are swift files, and I can't import objective C headers into Swift headers, except by adding them here.